general_parameters:
    qualisys_topic: /qualisys/mur620b/pose
    mir_topic: /mur620b/robot_pose
    good_tranformations: 50
    publish_transformation: false
    use_median: false
    print_all_transformations: false